android/hybridcommon-ui/src/main/java/com/revenuecat/purchases/hybridcommon/ui/PaywallFragment.kt

```diff
@@ -45,7 +45,15 @@ internal class PaywallFragment : Fragment(), PaywallResultHandler {
         get() = arguments?.getString(requiredEntitlementIdentifierKey)
 
     private val shouldDisplayDismissButton: Boolean?
-        get() = arguments?.getBoolean(shouldDisplayDismissButtonKey)
```

Comment:
Does using a default value of null not work in this case?

`get() = arguments?.getBoolean(shouldDisplayDismissButtonKey, null)`

Reply:
ahhh that would be easier lol, I will make the update

Reply:
Actually, not, the default value needs to be a boolean so that won't work